scripts/datasets/information_need_annotation/information_need_prompt_default.txt

```diff
@@ -0,0 +1,18 @@
+Given the annotated dialogue below, your task is to identify the information need of the user. The information need corresponds to what the user is looking for in the dialogue. The information need comprises constraints and requests. Constraints are conditions that the recommended movie must satisfy. Requests are properties of the movie that the user inquires about in the dialogue.
```

Comment:
Nit: "identify the information need" => "identify the underlying information need"

Reply:
Done.